test/util/fixtures/ClientServiceFixtures.java

```diff
@@ -0,0 +1,61 @@
+package util.fixtures;
+
+import entities.Client;
+import enums.Roles;
+import enums.StatusEnum;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class ClientServiceFixtures {
+
+  private static Client client1 = new Client();
+  private static Client client2 = new Client();
+  private static Client client3 = new Client();
+  private static Client client4 = new Client();
+  public static final Long FAKEID = Long.valueOf(10);
+
+  public static Client getClient1() {
+    buildClient(client1, Roles.CLIENT, 21, "Diana", "dxmd", "Monegro", StatusEnum.ACTIVE);
+    return client1;
+  }
+
+  public static Client getClient2() {
+    buildClient(client3, Roles.CLIENT, 22, "Dania", "dianamd", "Monegro", StatusEnum.DELETED);
+    return client2;
+  }
+
+  public static Client getClient3() {
+    buildClient(client1, Roles.CLIENT, 30, "Severino", "sevmonegro", "Monegro", StatusEnum.ACTIVE);
+    return client3;
+  }
+
+  public static Client getClient4() {
+    buildClient(client1, Roles.CLIENT, 40, "Angelina", "angie", "Peralta", StatusEnum.ACTIVE);
+    client4.setId(0L);
+    return client4;
+  }
+
+  public static List<Client> getClientsList() {
```

Comment:
Rename to `buildClients` and send the amount of clients that will be created by parameter.

Reply:
solved